src/test/java/resources/TwitterRequestResourceTest.java

```diff
@@ -67,4 +67,23 @@ public void invalidPost() throws TwitterException {
         Response response = twitterRequestResource.postTweet(new Message("Hello!"));
         verify(mockedHandler, times(1)).ResponseBuilder(any(Exception.class));
     }
+
+    /*ADDITIONAL TESTS*/
```

Comment:
no need to say additional tests, it won't make sense to anyone else if you collab on a codebase